test/unit/foreman/http_proxy_test.rb

```diff
@@ -1,6 +1,6 @@
 require 'test_helper'
 
-class HttpProxyTest < ActiveSupport::TestCase
+class Foreman::HttpProxyTest < ActiveSupport::TestCase
```

Comment:
Why does this need to be in the `Foreman` namespace? Because it's in the `foreman` directory in tests?